packages/lib/src/components/styles/fonts.ts

```diff
@@ -1,5 +1,11 @@
 import { css } from "@emotion/react";
 
+/**
+ * Definitions for font faces.
+ *
+ * WOFF2 files use an absolute path because they're used for the actual application.
+ * WOFF files use a relative path because they're only used for Storybook, which is less important.
```

Comment:
Maybe we should just create a separate stylesheet for storybook? It would probably be more proper to separate our concerns, since it can just go in the storybook package